specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -8492,37 +9449,54 @@
           "description": "The time (in UTC) when the last RPO value was calculated by Protection Service.",
           "type": "string"
         },
+        "primaryAvailabilityZone": {
+          "description": "The primary availability zone.",
+          "type": "string"
+        },
         "recoveryAvailabilityZone": {
           "description": "The recovery availability zone.",
           "type": "string"
         },
         "vmEncryptionType": {
```

Comment:
please correct this in code

Reply:
added in onenote

---
In reply to: [592114339](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r592114339) [](ancestors = 592114339)